python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -180,6 +183,50 @@ def replace_illegal_columns_chars(char: str, database: BaseDatabase) -> str:
 
         return normalize_config
 
+    def get_openlineage_facets(self) -> OpenLineageFacets:
+        """
+        Returns the lineage data
+        """
+        input_files = resolve_file_path_pattern(
+            self.input_file.path,
+            self.input_file.conn_id,
+            normalize_config={},
+            filetype=self.input_file.type.name,
+        )
+
+        input_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=self.input_file.openlineage_dataset_namespace,
+                name=self.input_file.openlineage_dataset_name,
+                facets={
+                    "file_size": self.input_file.size,
+                    "is_pattern": self.input_file.is_pattern(),
+                    "files": [file.path for file in input_files],
+                    "number_of_files": len(input_files),
+                },
+            )
+        ]
+
+        output_database = create_database(self.output_table.conn_id)
+        output_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=output_database.openlineage_dataset_namespace(),
+                name=output_database.openlineage_dataset_name(table=self.output_table),
+                facets={
+                    "metadata": self.output_table.metadata,
+                    "columns": self.output_table.columns,
+                    "schema": self.output_table.sqlalchemy_metadata.schema,
```

Comment:
Same comment about facets. Schema should be standarized `SchemaDatasetFacet`: https://github.com/OpenLineage/OpenLineage/blob/main/spec/facets/SchemaDatasetFacet.json

Reply:
I have created a custom facet here: https://github.com/astronomer/astro-sdk/pull/1025/commits/fd81ed87cbc39ba16cc4d030dafa55a86223f65a